watertap/property_models/NaCl_prop_pack.py

```diff
@@ -741,49 +741,34 @@ def calculate_scaling_factors(self):
             iscale.set_scaling_factor(self.enth_flow,
                                       iscale.get_scaling_factor(self.flow_mass_phase_comp['Liq', 'H2O'])
                                       * iscale.get_scaling_factor(self.enth_mass_phase['Liq']))
-
         # transforming constraints
-        # property relationships with no index, simple constraint
-        v_str_lst_simple = ('osm_coeff', 'pressure_osm')
-        for v_str in v_str_lst_simple:
-            if self.is_property_constructed(v_str):
-                v = getattr(self, v_str)
-                sf = iscale.get_scaling_factor(v, default=1, warning=True)
-                c = getattr(self, 'eq_' + v_str)
-                iscale.constraint_scaling_transform(c, sf)
+        if self.is_property_constructed('pressure_osm'):
+            sf = iscale.get_scaling_factor(self.pressure_osm, default=1, warning=True)
+            iscale.constraint_scaling_transform(self.eq_pressure_osm, sf)
+        if self.is_property_constructed('osm_coeff'):
+            sf = iscale.get_scaling_factor(self.mass_frac_phase_comp['Liq', 'NaCl'], default=1, warning=True)
+            iscale.constraint_scaling_transform(self.eq_osm_coeff, sf**1.25)
```

Comment:
This looks really odd to me. The osmotic coefficient should really just have a scaling factor of 1, the value ranges from 0.8 to 1.3. It shouldn't be tied to the scaling factor of the mass fraction of NaCl, and it shouldn't be raised to a power of 1.25.